src/Symfony/Component/Serializer/Tests/SerializerTest.php

```diff
@@ -842,6 +847,44 @@ public function testTrueBuiltInTypes()
         $this->assertEquals(new TrueBuiltInDummy(), $actual);
     }
 
+    public function testDeserializeUntypedFormat()
```

Comment:
Is it really needed to test the whole deserialization instead of only the denormalization?

If yes, the test might be located in the `AbstractObjectNormalizerTest`